numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2860,6 +2860,42 @@ def add_newdoc(place, name, doc):
 
     """)
 
+add_newdoc('numpy.core.umath', 'divmod',
+    """
+    Return element-wise floor division and remainder simultaneously.
+
+    Like ``(np.floor_divide(x, y), np.remainder(x, y))``, but faster because it
+    avoids redundant work. Equivalent to Python built-in divmod.
+
+    Parameters
+    ----------
+    x1 : array_like
+        Dividend array.
+    x2 : array_like
+        Divisor array.
+    out : tuple of ndarray, optional
+        Arrays into which the output is placed. Their types are preserved and
+        must be of the right shape to hold the output. See doc.ufuncs.
```

Comment:
Note that `See doc.ufuncs` doesn't actually produce a link of any kind, so should probably not be there at all, to avoid furthering the myth that it works.

Once we merge #9026, we'll get those links for every ufunc anyway.